proto/binlogdata.proto

```diff
@@ -476,6 +478,10 @@ message MinimalSchema {
   repeated MinimalTable tables = 1;
 }
 
+message VStreamOptions {
+  repeated string internal_tables = 1;
+}
+
```

Comment:
should we also pass on the list of internal tables to stream? Or Filter is the way to provide?
The ask could be to stream all tables from the main database but stream specific tables from sidecar db tables.

Reply:
I see this is a slice of string so, this would be the list of internal tables and filter is separate.

Reply:
As discussed separately, we are currently using internal tables in the internal VTTablet VStreamer API. The support at the VTGate VStream API level doesn't specify an implementation. Currently we use the `_vt.heartbeat` to support it, but we might do it in a different way in the future if indicated.